### DISCUSSION TITLE
Please add a config option to switch breakpoints from mobile first to desktop first

### DISCUSSION THREAD

**Discussion by caotrem** [chakra-ui/panda#2442] [Ideas]
Hey there!

I'm missing an option in the config to switch the currently used mobile first (min-width) approach to desktop first (max-width).
Currently there seems to be no way to switch this:
https://panda-css.com/docs/concepts/responsive-design

I really love PandaCSS. But this makes it (unnecessary) hard to convert our current style system based on desktop first.

**Comment by astahmer** [comment]
is this what you're looking for ? these are auto-generated based on your breakpoints
![image](https://github.com/chakra-ui/panda/assets/47224540/463c9604-212f-4346-aa48-9da36dda4a18)

**Comment by caotrem** [comment]
Yes, great! That does the trick for me. Thank you.
I think it would be helpful to adjust the docs. They are misleading here:

https://panda-css.com/docs/concepts/responsive-design
<img width="786" alt="image" src="https://github.com/chakra-ui/panda/assets/47040897/942c5bc0-468f-454f-81cf-c99c28c39136">

And do not mention the  **Down options anywhere under "Responsive Design". It sounds like ONLY mobile first is supported.

**Reply by astahmer** [reply]
would you like to send a PR ? docs file [is here](https://github.com/chakra-ui/panda/blob/eb8a238889ba4d728a281c278c616068455ac8c8/website/pages/docs/concepts/responsive-design.md#L25)
